build/azure-pipelines-yarn-template.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       node_10_x:
         node_version: 10.x
       node_8_x:
-        node_version: 8.x
+        node_version: ">=8.16.0"
```

Comment:
I wondered that it does not trigger using of node 12 😕 

12.0.0 satisfies constraint ">=8.16.0"

Reply:
no sorry it's important to ensure that our CI builds even with Node.js 8.x because for Fastify v3 that will be minimum supported release (but 8.x on Azure currently doesn't work because it uses 8.10.0); anyway we have other builds with 10.x and 12.x ... @L2jLiga is it ok even for you ?

Reply:
I mean, that using `^8.16.0` would be even better

Reply:
Or maybe just `8.16.0`

Reply:
I'll try asap with '^8.16.0' so in case of other 8.x versions we should be fine, thanks for the hint

Reply:
done, it seems to work, I agree that with your suggestion looks better